articles/site-recovery/avs-tutorial-failover.md

```diff
@@ -74,10 +74,10 @@ Verify properties as follows:
 
 In some scenarios, failover requires additional processing that takes around 8 to 10 minutes to complete. You might notice longer test failover times for:
 
-* VMware VMs running a Mobility service version older than 9.8.
-* VMware Linux VMs.
-* VMware VMs that don't have the DHCP service enabled.
-* VMware VMs that don't have the following boot drivers: storvsc, vmbus, storflt, intelide, atapi.
+* VMware vSphere VMs running a Mobility service version older than 9.8.
```

Comment:
Please use "VMware VMs" in all places.

Reply:
@Sharmistha-Rai - unfortunately VMware VMs is a misnomer and we need to follow the naming guidelines of VMware. VMware vSphere is VMware product that runs Virtual Machines